commands/handbook.py

```diff
@@ -0,0 +1,70 @@
+from commands.base import Command
+from discord import Embed
+
+import bs4, re, requests
+
+def subject_details(code):
+	page = requests.get('https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code)
+
+	if page.status_code != 200:
+		page = requests.get('https://www.handbook.unsw.edu.au/postgraduate/courses/2019/' + code)
+
+		if page.status_code != 200:
+			return None
+
+	soup = bs4.BeautifulSoup(page.text, features='lxml')
+
+	course_title = re.sub(r'^\s+|\s+$', '', soup.find('span', attrs={'data-hbui' : 'module-title'}).string)
```

Comment:
Instead of substituting whitespace can't you use `str.strip()`? Same with line below